tests/Data/ValidPayPalNotificationRequest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Data;
+
+use WMDE\Fundraising\Frontend\UseCases\HandlePayPalPaymentNotification\PayPalNotificationRequest;
+use WMDE\Fundraising\Frontend\Domain\Model\Euro;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class ValidPayPalNotificationRequest {
```

Comment:
Such copyright infringement. This is totally MY pattern. MINE MINE MINE MINE. You now owe me one lolcat in royalties